lib/views/create-bug.js

```diff
@@ -230,12 +257,24 @@ module.exports = function(ctx) {
 
     newBug();
 
-    // Activity data overrides locally stored data for now
-    var activity = app.activity;
-    if (activity) {
-      addActivityAttachments(activity.data.blobs, activity.data.filenames);
-      app.activity = null;
-    } else if (data) {
+    if (app.activity) {
+      var activity = app.activity;
+      var dialog = form.querySelector('#pickAttachments');
+      var list = form.querySelector('#attachments');
+      var attachForm = form.querySelector('#pickAttachmentsForm');
+      activity.data.blobs.forEach(function(blob, i) {
+        list.appendChild(createAttachmentRow(activity.data.filenames[i]));
+      });
+      attachForm.addEventListener('submit', function(e) {
+        e.preventDefault();
+        var checked = attachForm.querySelectorAll('input:checked');
+        var files = [].map.call(checked, function(e) { return e.value; });
+        addActivityAttachments(activity.data.blobs, activity.data.filenames, files);
+        app.activity = null;
+        dialog.style.display = 'none';
```

Comment:
You should use a ```[hidden]``` attribute or ```.hide``` class instead of manually setting styles like this.